bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -31,6 +31,7 @@ class Neuron(CMakePackage):
     patch("patch-v800-cmake-nvhpc.patch", when="@8.0.0%nvhpc^cmake@3.20:")
 
     version("develop", branch="master")
+    version("9.0.a6", commit="5665e19")
```

Comment:
```suggestion
    version("9.0.a6", commit="67a672a")
```
neuronsimulator/nrn@67a672a2c802cb57e8a216d80af4b046f808bb02 would include https://github.com/neuronsimulator/nrn/pull/2350 and https://github.com/neuronsimulator/nrn/pull/2259. That would give us a fighting chance of being able to deploy `%oneapi` without a new `neuron` version.

Reply:
Just to make the explicit link, this ought to mean that https://github.com/BlueBrain/spack/pull/1891 would not need to add a new `neuron` version.